src/main/java/org/mockito/MockitoSession.java

```diff
@@ -48,7 +48,9 @@
  *         //initialize session to start mocking
  *         mockito = Mockito.mockitoSession()
  *            .initMocks(this)
+ *            .name("Example")
```

Comment:
'name' and 'logger' are targeted to framework integrations rather than regular users. I'd rather we don't include them here in this example. This example is for regular users who want to use mockito session in their test. They won't know what name or logger to configure :)

I suggest to remove those 2 methods from this example completely. I don't think they need an example at the top level.